client/src/pages/CreateOrganizationProfile.js

```diff
@@ -105,11 +106,15 @@ const CreateOrgProfile = (props) => {
 
   const onFormSubmit = async (formData) => {
     if (!privacy) {
-      alert("You must agree to our privacy policy before proceeding");
-      return;
+      return createOrganizationFormDispatch({
+        type: CREATE_ORGANIZATION_ERROR,
+        error: `You must agree to our privacy policy before proceeding`,
+      });
     } else if (!conditions) {
-      alert("You must agree to our terms and conditions before proceeding");
-      return;
+      return createOrganizationFormDispatch({
+        type: CREATE_ORGANIZATION_ERROR,
+        error: `You must agree to our terms and conditions before proceeding`,
+      });
```

Comment:
These two could also be unnecessary or we should change a bit of the logic on the `Create Profile` button, thing is that the way that right now it is happening is that the button will not be enabled until the user checks both boxes. I think this approach of showing the error is better, so perhaps let's work around that?

Reply:
Also, check the Build URL, I am still getting this -> 
![image](https://user-images.githubusercontent.com/25408660/86508937-02619780-bdb2-11ea-9a13-16310791dad9.png)

Reply:
And this is because I am using the same email of the user / another org as well. I mean it is much better than it was / before we didn't even see anything , just the `create profile` button didn't do anything (looked like)